cala-ledger/src/velocity/mod.rs

```diff
@@ -157,18 +157,17 @@ impl Velocities {
         account_ids: &[AccountId],
         account_set_mappings: &HashMap<AccountId, Vec<AccountSetId>>,
     ) -> Result<(), VelocityError> {
-        let mut unique_ids = account_ids.iter().copied().collect::<HashSet<_>>();
```

Comment:
For this I was thinking there could be cases where 2 accounts in a transaction fall under the same account set in a chart of accounts for e.g., which would mean we'd get multiple instances of every account id in the chart tree and any statement trees that attach to the chart. Not sure if this is a concern for controlling the number of ids we eventually pass to the sql query?